src/lib/styles/global/menu.scss

```diff
@@ -0,0 +1,105 @@
+@use "../mixins/media";
+
+:root {
+  div.split-pane,
+  div.stretch-pane {
+    div[role="menu"]:first-child {
+      --menu-width: var(--menu-expanded-width);
+
+      button {
+        &.menu-collapse {
+          opacity: 0;
+          visibility: hidden;
+          transition:
+            opacity var(--animation-time-long),
+            transform var(--animation-time-normal);
+
+          padding: var(--padding-0_5x);
+          margin: 0 auto;
+
+          background: var(--card-background);
+          color: var(--tertiary);
+
+          width: var(--padding-4x);
+          height: var(--padding-4x);
+          min-height: var(--padding-4x);
+
+          transform: rotate(0deg);
+
+          display: none;
+
+          @media (min-height: 654px) {
```

Comment:
Is this some empirical value?

Reply:
Asking because I saw this value in nns-dapp too.

Reply:
Yes. I added comments now.

Reply:
And if it can be a css variable that could be nice. If not/not-elegant, then that's not a problem either.